model/distro/distro.go

```diff
@@ -695,6 +703,7 @@ func (d *Distro) GetResolvedPlannerSettings(s *evergreen.Settings) (PlannerSetti
 		MainlineTimeInQueueFactor: ps.MainlineTimeInQueueFactor,
 		ExpectedRuntimeFactor:     ps.ExpectedRuntimeFactor,
 		GenerateTaskFactor:        ps.GenerateTaskFactor,
+		StepbackTaskFactor:        ps.StepbackTaskFactor,
```

Comment:
We need to add an if statement below to use the admin factor if it's not defined for the distro (or, since we aren't adding this to the distro page at all, we could just not add it to planner settings and set it immediately to the admin setting)

Reply:
done